src/interfaces/wallet.h

```diff
@@ -132,6 +132,9 @@ class Wallet
     //! List locked coins.
     virtual void listLockedCoins(std::vector<COutPoint>& outputs) = 0;
 
+    // Whether this or any UTXO with the same CTxDestination has been spent.
```

Comment:
The comment is not clear to me, what you mean by "this"?

Reply:
this comment was carried over from the original `IsUsedDestination` method in the `wallet/wallet.h` file